wrappers/python/tests/anoncreds/test_prover_get_claims.py

```diff
@@ -1,40 +1,54 @@
 from indy.anoncreds import prover_get_claims
 from indy.error import ErrorCode, IndyError
 
-from tests.utils import anoncreds
-
 import json
 import pytest
 
 
+# noinspection PyUnusedLocal
 @pytest.mark.asyncio
-async def test_prover_get_claims_works_for_empty_filter(init_common_wallet):
-    claims = json.loads(await prover_get_claims(init_common_wallet[0], "{}"))
+async def test_prover_get_claims_works_for_empty_filter(wallet_handle, prepopulated_wallet):
+    claims = json.loads(
+        await prover_get_claims(wallet_handle, "{}"))
+
     assert len(claims) == 1
```

Comment:
Can we check that the returned claims match the expected result (as defined by API)?